### ISSUE TITLE
Office UI Fabric React FocusTrapZone : Aria-Hidden Elements are focusable

### ISSUE THREAD

**Issue by Aishwaryathota** [microsoft/fluentui#17916]
### Environment Information

- **Package version(s)**: (7.126.0, 8.11.2)
- **Browser and OS versions**: (Chrome Windows10)

### Describe the issue:

FocusTrapZone component is rendering elements which are aria-hidden but are focusable. 

### Please provide a reproduction of the issue in a codepen:

![image](https://user-images.githubusercontent.com/37802871/115684924-62dbb880-a375-11eb-968a-1c96f0fd196b.png)

#### Actual behavior:

FocusTrapZone is rendering elements which have aria-hidden-true but are focusable

#### Expected behavior:

For elements which have aria-hidden = true, elements should not be focusable. Focusable content should have tabindex='-1' or be removed from the DOM

### Documentation describing expected behavior
https://accessibilityinsights.io/info-examples/web/aria-hidden-focus/
https://dequeuniversity.com/rules/axe/4.0/aria-hidden-focus

**Comment by gouttierre** [comment]
@Aishwaryathota - thank you for filing this item with us. 

@miroslavstastny - Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley

**Comment by tianbli** [comment]
same as https://github.com/microsoft/fluentui/issues/16277 Are you going to change this at this moment?

**Comment by cvabanuk3** [comment]
@Aishwaryathota @miroslavstastny @smhigley 
cc: @khmakoto 

Just want to follow up on this issue, as this leads to Fast pass issues.
Definitely it's regression from below change, as we got aria-hidden as true in this under FocusTapZone.tsx file change, as we dont see this issue in prior versions, 
https://github.com/microsoft/fluentui/commit/5e0957143308b2d1a3b49d0085b4a06eaf300404

Fast pass issue observed:
Title: WCAG 1.3.1,WCAG 4.1.2: Ensures aria-hidden elements do not contain focusable elements (div[data-is-visible="true"][aria-hidden="true"]:nth-child(1))
Tags: Accessibility, WCAG 1.3.1, WCAG 4.1.2, aria-hidden-focus

Issue: Ensures aria-hidden elements do not contain focusable elements (aria-hidden-focus - https://accessibilityinsights.io/info-examples/web/aria-hidden-focus)

Target application: Test Form | Xbox Support - https://dev.support.xbox.com/en-US/forms/test

Element path: div[data-is-visible="true"][aria-hidden="true"]:nth-child(1)

Snippet: <div aria-hidden="true" tabindex="0" data-is-visible="true" style="pointer-events: none; position: fixed;"></div>

How to fix: 
Fix all of the following:
  Focusable content should have tabindex='-1' or be removed from the DOM

Environment: Microsoft Edge version 91.0.864.59

====

This accessibility issue was found using Accessibility Insights for Web 2.27.0 (axe-core 4.2.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by khmakoto** [comment]
@cvabanuk3 as Sarah explains in #16277 this is an automated error and not a practical one. Given that I don't think we are going to be updating this anytime soon.

**Comment by SalmanSaberi** [comment]
Hi team
Please let us know if this is a "Wont Fix" issue or gives us an ETA by when this issue can be closed ?
